modules/combined/test/tests/phase_field_fracture/crack2d_aniso_cleavage_plane.i

```diff
@@ -0,0 +1,197 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 40
+  ny = 20
+  ymax = 0.5
+[]
+
+[MeshModifiers]
+  [./noncrack]
+    type = BoundingBoxNodeSet
+    new_boundary = noncrack
+    bottom_left = '0.5 0 0'
+    top_right = '1 0 0'
+  [../]
+[]
+
+[GlobalParams]
+  displacements = 'disp_x disp_y'
+[]
+
+[Variables]
+  [./c]
+    family = LAGRANGE
+    order = FIRST
+  [../]
+[]
+
+[Modules]
+  [./TensorMechanics]
+    [./Master]
+      [./All]
+        add_variables = true
+        strain = SMALL
+        additional_generate_output = 'strain_yy stress_yy'
+        planar_formulation = PLANE_STRAIN
+      [../]
+    [../]
+  [../]
+[]
+
+[Kernels]
+  [./ACbulk]
+    type = AllenCahn
+    variable = c
+    f_name = F
+  [../]
+  [./ACInterfaceCleavageFracture]
+    type = ACInterfaceCleavageFracture
+    variable = c
+    beta_penalty = 1
```

Comment:
minor issue, is beta_penalty = 1 sufficient? I thought it has to be very large.

Reply:
beta=1 works well in cases where Gc ~ 2 J/m^2. If Gc is very high, then higher values of beta are required. The value in test input file is place holder.